Cargo.toml

```diff
@@ -23,5 +23,10 @@ default = ["std", "unlimited-points"]
 [badges]
 travis-ci = { repository = "daboross/rust-throw" }
 
+[dependencies]
+serde={version = "1.0", default-features = false }
+serde_derive = "1.0"
```

Comment:
It would be nice to make these optional dependencies. Even with no_std, `serde` is an additional crate which we shouldn't be needed unless someone else wants to serialize `throw` structures. [`#[cfg_attr]`](https://chrismorgan.info/blog/rust-cfg_attr.html) could be useful for doing this if also doing derives.

If we do want to do this, I would suggest a `serde-1` feature which depends on both `serde` and `serde_derive`.

Reply:
@daboross the feature tag `serde-1` was added, is not activated by default, also a new test to the travis file was added `cargo test --features serde-1`